packages/cli/src/index.ts

```diff
@@ -503,6 +505,16 @@ program
 
 const pluginCmd = program.command('plugin').description('Manage Cannon plug-in modules');
 
+pluginCmd
+  .command('list')
+  .description('List all installed plugins')
+  .action(async function () {
+    console.log(green(bold('\n=============== Installed Plugins ===============')));
+    const installedPlugins = await listInstalledPlugins();
+    installedPlugins.forEach((plugin) => console.log(yellow(plugin)));
+    console.log('Complete!');
```

Comment:
I think we can probably remove this line, otherwise lgtm!

Reply:
@noahlitvin  Which line do you mean( you have selected four lines), I assume you mean `console.log('Complete!');` Am I right?

Reply:
yes that line